bin/server.js

```diff
@@ -1,62 +1,52 @@
-#!/usr/bin/env node
+#!usr/bin/env node
+'use strict'
 var base    = require('taskcluster-base');
 var path    = require('path');
 var debug   = require('debug')('events:bin:server');
-var Promise = require('promise');
 var socket  = require('../events/socket');
 var express = require('express');
+var path    = require('path');
+var assert  = require('assert');
 
-/** Launch server */
-var launch = function(profile) {
-  // Load configuration
+var launch = (profile,env) => {
   var cfg = base.config({
-    defaults:     require('../config/defaults'),
-    profile:      require('../config/' + profile),
-    envs:         [
-      'influx_connectionString',
-      'pulse_username',
-      'pulse_password'
+    files: [
+      path.join(__dirname,'../config.yml')
```

Comment:
look at the defaults:
https://github.com/taskcluster/typed-env-config/blob/master/src/config.js#L48-L55

Just make sure current working folder is the top-level directory...

note that: `process.env.NODE_ENV` is "production" in heroku. so we only really need to pass it in when we're in testing...

What is the use for overwriting env... I think we can avoid that..